src/main/tray.ts

```diff
@@ -0,0 +1,104 @@
+import path from "path"
+import sharp from "sharp";
+import packageInfo from "../../package.json"
+import { dialog, Menu, nativeImage, Tray } from "electron"
+import { isDevelopment, isMac } from "../common/vars";
+import { autorun } from "mobx";
+import { showAbout } from "./menu";
+import { AppUpdater } from "./app-updater";
+import { WindowManager } from "./window-manager";
+import { clusterStore } from "../common/cluster-store";
+import { workspaceStore } from "../common/workspace-store";
+import { preferencesURL } from "../renderer/components/+preferences/preferences.route";
+import { clusterViewURL } from "../renderer/components/cluster-manager/cluster-view.route";
+import logger from "./logger";
+
+// note: instance of Tray should be saved somewhere, otherwise it disappears
+export let tray: Tray;
+
+export const trayIcon = isDevelopment
+  ? path.resolve(__static, "../src/renderer/components/icon/logo-lens.svg")
+  : path.resolve(__static, "logo.svg") // electron-builder's extraResources
+
+export function initTray(windowManager: WindowManager) {
+  return autorun(() => {
+    const menu = createTrayMenu(windowManager);
+    buildTray(menu);
+  })
+}
+
+export async function buildTray(menu: Menu) {
+  logger.info("[TRAY]: build start");
+  const iconSize = isMac ? 16 : 32; // todo: verify on windows/linux
+  const pngIcon = await sharp(trayIcon).png().toBuffer();
```

Comment:
We already have png icons in build dir, can we use those?

Reply:
Yes, we could, but it  has different design (more heavy imho) and it's colourful (might better match for linux/win through).

![image](https://user-images.githubusercontent.com/6377066/94803734-6c5dfd00-03f2-11eb-9ab0-dd1a4ce52ab7.png)

@jakolehm @miskun wdyt ^?

Reply:
with `.grayscale()`:
![image](https://user-images.githubusercontent.com/6377066/94847201-f033db80-042a-11eb-8a78-33b3d07e7e5d.png)